packages/fetchai/protocols/acn/README.md

```diff
@@ -0,0 +1,76 @@
+# ACN Protocol
+
+## Description
+
+This is a protocol for ACN (agent communication network) envelope delivery.
+
+## Specification
+
+```yaml
+---
+name: acn
+author: fetchai
+version: 1.0.0
+description: The protocol used for envelope delivery on the ACN.
+license: Apache-2.0
+aea_version: '>=1.0.0, <2.0.0'
+protocol_specification_id: aea/acn:1.0.0
+speech_acts:
+  register:
+    record: ct:AgentRecord
+  lookup_request:
+    agent_address: pt:str
+  lookup_response:
+    record: ct:AgentRecord
+  aea_envelope:
+    envelope: pt:bytes
+    record: ct:AgentRecord
+  status:
+    body: ct:StatusBody
+...
+---
+ct:AgentRecord:
+  string service_id = 1;
+  string ledger_id = 2;
+  string address = 3;
+  string public_key = 4;
+  string peer_public_key = 5;
+  string signature = 6;
+  string not_before = 7;
+  string not_after = 8;
+ct:StatusBody: |
+  enum StatusCodeEnum {
+    // common (0x)
+    SUCCESS = 0;
```

Comment:
@DavidMinarsch I thought you agreed on using three digits instead of two (I have seen a thumbs up in another comment). Just to double-check whether you rejected the suggestion, or you forgot it.

Reply:
https://github.com/fetchai/agents-aea/pull/2447#discussion_r611033668